Sources/Compositor/OwlShmBuffer.m

```diff
@@ -168,7 +168,7 @@ - (NSSize) size {
     return NSMakeSize(width, height);
 }
 
-- (void) drawInRect: (NSRect) rect {
+- (void) drawInRect: (CGRect) rect {
```

Comment:
This method is defined as taking an `NSRect`, see `OwlBuffer.h`. Instead, we need to convert the `NSRect` argument into a `CGRect` before passing it into `CGContextDrawImage`.